Cargo.toml

```diff
@@ -14,6 +14,7 @@ edition = "2018"
 byteorder = "1"
 ff_derive = { version = "0.4.0", path = "ff_derive", optional = true }
 rand_core = "0.5"
+zeroize = {version = "1.1", features = ["zeroize_derive"]}
```

Comment:
I'm still not sure that we want to have a required dependency on `zeroize` for the `ff` crate, especially as `Zeroize` is not part of any bound on the `Field` or `PrimeField` traits. I think I would prefer that we make this crate optional, and instead of using `zeroize_derive` we manually add an `impl Zeroize for #repr` if the `zeroize` feature is enabled.

This would require adding a `zeroize` feature flag to `ff_derive`, and then here:
```
[dependencies]
...
zeroize_crate = { package = "zeroize", version = "1.1", optional = true }

[features]
...
zeroize = ["ff_derive/zeroize", "zeroize_crate"]
```
([Crates that use `namespaced-features` can't be published](https://github.com/rust-lang/api-guidelines/issues/180#issuecomment-430485651), so we can't have a feature with the same name as a crate.)

cc @tarcieri.

Reply:
We make `zeroize` optional in most of the RustCrypto crates, FWIW.